soap/src/java/com/zimbra/soap/mail/message/GetPop3UIDRequest.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * ***** BEGIN LICENSE BLOCK *****
+ * Zimbra Collaboration Suite Server
+ * Copyright (C) 2019 Synacor, Inc.
```

Comment:
Check other files as well.
```suggestion
 * Copyright (C) 2023 Synacor, Inc.
```

Reply:
Done